Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,81 @@
+/*Global Styles*/
+
+body{
+	background-color:#F6F6F6;
+	padding: 75;
+	border-bottom: 5px solid #000;
+	border-top: 5px solid #000;
+	font-family: 'Lora', serif;
+}
+
+h1{
+	font-family: sans-serif;
```

Comment:
Looks like you have a generic `font-family: sans-serif;` _plus_ a specific font declaration on the same element in a few spots — this might be something you added at first and forgot to remove, but just note that a property can only be read once per selector, and if it's duplicated it will favor the one that appears later in the declaration, so it worked out fine in this case, but you'd want to remove those duplicate declarations for sake of clarity.